core/src/main/java/org/elasticsearch/common/network/NetworkModule.java

```diff
@@ -381,6 +395,37 @@ public void registerTaskStatus(String name, Writeable.Reader<? extends Task.Stat
         namedWriteableRegistry.register(Task.Status.class, name, reader);
     }
 
+    /**
+     * Register an allocation command.
+     * <p>
+     * This lives here instead of the more aptly named ClusterModule because the Transport client needs these to be registered.
```

Comment:
Oh! You are suggesting a new Module just for allocation commands. I think folks are generally against new Modules and like to remove modules. That is why NetworkModule and SearchModule are so large.